packages/web-components/src/components/rux-progress/test/index.html

```diff
@@ -21,5 +21,22 @@
 
     <body>
         <rux-progress max="100" min="0" value="50"></rux-progress>
+        <ftl-belt access-token="" file-id="">
+            <ftl-holster name="Progress Empty" node="24958:114399">
+                <div style="width: 381px">
+                    <rux-progress max="100" min="0" value="0"></rux-progress>
+                </div>
+            </ftl-holster>
+            <ftl-holster name="Progress Half" node="25864:114700">
+                <div style="width: 390px">
+                    <rux-progress max="100" min="0" value="50"></rux-progress>
+                </div>
+            </ftl-holster>
+            <ftl-holster name="Progress Full" node="25864:114706">
+                <div style="width: 399px">
+                    <rux-progress max="100" min="0" value="100"></rux-progress>
+                </div>
+            </ftl-holster>
+        </ftl-belt>
```

Comment:
comment out

Reply:
Sure thing! Good note. I thought it was just the call to the holster script that blew up tests.